html/pfappserver/root/authentication/source/type/SAML.tt

```diff
@@ -0,0 +1,18 @@
+
+[% form.field('sp_entity_id').render | none %]
+[% form.field('sp_key_path').render | none %]
+[% form.field('sp_cert_path').render | none %]
+[% form.field('idp_metadata_path').render | none %]
+[% form.field('idp_entity_id').render | none %]
+[% form.field('idp_cert_path').render | none %]
+[% form.field('idp_ca_cert_path').render | none %]
+[% form.field('username_attribute').render | none %]
+[% form.field('authorization_source_id').render | none %]
```

Comment:
They could but that would be the only authentication source to work this way...

I think that it is better to follow the standard of the other auth sources so it is easier to follow and search when we have to dig through the code